### DISCUSSION TITLE
Simple drawing & basic markdown

### DISCUSSION THREAD

**Discussion by Xiarno** [rbreu/beeref#38] [Ideas & General Discussion]
It'll be very useful to be able to do simple drawing & basic markdowns.
• Simple drawing like, just drawing simple shapes with different colors (or/and free drawing too). Shapes like squares, circles & arrows.
• Basic markdown like the ability to make the text (And more importantly part of the text, not all the block at once) bold, italic, underline, and different colors.

This would launch this program a lot, and would help for a lot of things... I'm mainly talking about note-taking. I'm a very "note-taking" guy that just takes notes all the time. Using screenshots helps me even more since I'm visual. The two apps that I found could do that the best are Pureref and Beeref. However, even if both have amazing images capability, neither have the function I just suggested. I know that, from Pureref standpoint, it's been a highly requested feature for many years but never saw any implementation. With the latest update of Pureref being more than a year old and them being unresponsive, I've turned my attention to Beeref.
When I say that if this is added that it'll become one of my most used programs in my daily life, I am not kidding. Simply saying that I would need this to massively help me in my daily life is an understatement.

Please add this! This would be amazingly useful!

**Comment by tanzuha** [comment]
yeah it will be very usefull

**Comment by Cinderflame-Linear** [comment]
I have a PR up for drawing basic free-hand sketches. Hopefully it gets accepted :pray: 

https://github.com/rbreu/beeref/pull/150

**Comment by tanzuha** [comment]
Sure
